radis/api/tools.py

```diff
@@ -287,10 +293,12 @@ def replace_PQR_with_m101(df):
         raise NotImplementedError(df_type)
 
     # Then do the actual mapping
-    if df.dtypes["branch"] != np.int64:
+    if df.dtypes["branch"] != np.float64:
```

Comment:
what was the reason for this again? use of np.nan I believe? Would be better to keep branch np.int8 dtype. np.float64 uses 8x more memory

Reply:
I'll mark this as resolved, but would be good to open an issue on this if it doesn't already exist